test/fileImport.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const _ = require('lodash');
+const config = require('config');
+const should = require('should');
+const homedir = require('homedir');
+
+const client = require('./lib/client.js').client;
+const nav = require('./lib/nav/navigation.js')(client, config);
+const micis = require('./lib/auth/micis.js')(client);
+
+describe('Doing various imports', function() {
```

Comment:
Change `Doing various imports`  --> `Perform various imports and verify that they function correctly`